sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.controller.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable, NotFoundException } from "@nestjs/common";
+import { ApplicationOfferingChangeRequestService } from "../../services";
+import { ApplicationOfferingChangesAPIOutDTO } from "./models/application-offering-change-request.dto";
+import { getUserFullName } from "../../utilities";
+
+@Injectable()
+export class ApplicationOfferingChangeRequestControllerService {
+  constructor(
+    private readonly applicationOfferingChangeRequestService: ApplicationOfferingChangeRequestService,
+  ) {}
+
+  /**
+   * Get the Application Offering Change Request by its id.
+   * @param id the Application Offering Change Request id.
+   * @param options method options:
+   * - `studentId`: student id for student authorization.
+   * - `locationId`: location id for institution authorization.
+   * @returns application offering change request.
+   */
+  async getById(
+    id: number,
```

Comment:
it will be easy to understand, if you can name it was `applicationOfferingChangeRequestId`

Reply:
The reason I used `getById` was because the part `application-oFfering-change-request` was implied from the file name.

Reply:
comment was regarding line 21

Reply:
In most of places we have used in the way how @ann-aot had mentioned. For the sake of consistency we can follow it.

Reply:
https://github.com/bcgov/SIMS/pull/2196#discussion_r1306170921